.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 *.log
 tags
 .DS_Store
+.tern-project
```

Comment:
Interesting stuff! Maybe better to put tools we use that aren't a part of the project in a `.gitignore` in our own home directory though. Though I'm not very bothererd about it.